infrastructure/stacks/network/route53.tf

```diff
@@ -0,0 +1,14 @@
+resource "aws_route53_zone" "private" {
+  name = "uec-dos-mig-dev"
+  vpc {
+    vpc_id = aws_vpc.main.id
+  }
+}
+
+resource "aws_route53_record" "dev" {
+  zone_id = aws_route53_zone.private.zone_id
+  name    = "mig-datastore"
+  type    = "CNAME"
+  ttl     = 300
```

Comment:
Not sure what an appropriate ttl would be in this case so lets leave it as is for now unless other reviewer has an opinion

Reply:
the ttl is a requirement if using the "records" argument.